site/docs/encoding/legend.md

```diff
@@ -89,19 +89,19 @@ To provide themes for all legends, the legends config (`config: {legend: {...}}`
 
 ### General
 
-{% include table.html props="cornerRadius,fillColor,orient,offset,strokeColor,strokeDash,strokeWidth,padding" source="LegendConfig" %}
+{% include table.html props="columns,columnPadding,cornerRadius,fillColor,gridAlign,orient,offset,rowPadding,strokeColor,strokeDash,strokeWidth,padding" source="LegendConfig" %}
```

Comment:
A lot of these are now repeated.  Better not repeat them but instead say config support all properties except a, b, c...  (and, if applicable, config can also have a few more properties).